src/algorithm/entities/crane_work_area_type.rs

```diff
@@ -0,0 +1,27 @@
+use sal_sync::services::entity::error::str_err::StrErr;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+///
+/// Represents [crane work area types](design\docs\algorithm\part01\initial_data.md)
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
```

Comment:
If type equals to simple type, add `Copy` to derive to avoid .clone() calling:
```rast
let work_area1 = CraneWorkArea::Aggressive;
let work_area2 = work_area1;
```